Poker.BE/Poker.BE.Domain/Game/Card.cs

```diff
@@ -29,26 +34,28 @@ public enum Value
 		King,
 		Ace,
 	}
-	public class Card
+	public class Card : IComparable<Card>
     {
 		// TODO: complete - set team member to do this
 		#region Properties
 		protected Suit CardSuit { get; }
 		protected Value CardNumber { get; }
-		protected int ShuffledIndex { get; }//used for shuffling cards
+		protected int ShuffledIndex { get; set; }//used for shuffling cards
```

Comment:
if you only write to `ShuffledIndex` in this class, you should use `private set;` instead. that way you don't expose the `set` as `protected`